www-apps/radicale/radicale-2.1.9.ebuild

```diff
@@ -0,0 +1,84 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+PYTHON_COMPAT=( python{3_4,3_5,3_6} )
+
+inherit distutils-r1 eutils user
+
+MY_PN="Radicale"
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="A simple CalDAV calendar server"
+HOMEPAGE="https://radicale.org/"
+SRC_URI="mirror://pypi/${MY_PN:0:1}/${MY_PN}/${MY_P}.tar.gz"
+
+LICENSE="GPL-3"
```

Comment:
```
# the Free Software Foundation, either version 3 of the License, or
# (at your option) any later version.
```

That's GPL-3+.

Reply:
Not so sure ...
```$ md5sum COPYING /usr/portage/licenses/GPL-3
d32239bcb673463ab874e80d47fae504  COPYING
d32239bcb673463ab874e80d47fae504  /usr/portage/licenses/GPL-3```

Reply:
I am not sure i get that so please tell me exactly how to fix that. Do i set "GPL-3+" instead or do i go for an and/or list. And would that change also affect the other ebuilds, so i would have a first commit changing the license of them all and bumping to -r+1 followed by the bump ebuild?

Reply:
Just change it to GPL-3+ and you're done. No need to bump for a minor change like that (the original license was narrower, so there's no danger associated with keeping the old version).